source/Sashimi.AzureServiceFabric/Endpoints/AzureServiceFabricClusterEndpoint.cs

```diff
@@ -48,7 +47,7 @@ public override IEnumerable<Variable> ContributeVariables()
             }
         }
 
-        public string DefaultWorkerPoolId { get; set;  }
+        public string DefaultWorkerPoolId { get; set; } = string.Empty;
```

Comment:
Why default these to empty string?

Reply:
I wanted to remove the nullable disabling, which meant setting defaults for these to keep the compiler happy